### PR TITLE
bug/9119-Chika-NotLoggingVamaLoginTokenFetchErrors

### PR DESCRIPTION
**Pull request by cadibemma**
## Description of Change

- We have not logged a single `vama_login_token_fetch` error in production. It seems far more likely that our assumptions/logic behind how we could log that error are broken
## Testing

- [ ] Tested on iOS 
- [ ] Tested on Android 

## Reviewer Validations

- [ ] We should be logging those errors, with whatever related information would also be helpful, in production
## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

**Review by theodur — CHANGES_REQUESTED** [review]
I noticed the logic for the `vama_login_token_fetch` event wasn't updated. That one will need the same changes you made to `vama_login_token_refresh`.

**Comment by dumathane** [comment]
I think we can also just retire vama_exchange_failed as well.

File: VAMobile/src/constants/analytics.ts [hunk 1]
```diff
@@ -761,11 +761,11 @@ export const Events = {
       },
     }
   },
-  vama_login_token_refresh: (status_code: number): Event => {
+  vama_login_token_refresh: (error: Error): Event => {
     return {
       name: 'vama_login_token_refresh',
       params: {
-        status_code,
+        error,
```
Comment (theodur) [inline VAMobile/src/constants/analytics.ts:768]: I believe the error also needs to be converted to a string

Comment (cadibemma) [reply VAMobile/src/constants/analytics.ts]: converted!

**Review by theodur — APPROVED** [review]

**Comment by TKDickson** [comment]
@theodur @cadibemma have either/both of you built the app in both platforms with these changes and logged in? If not, I can add to my list (that's the basic stuff I want to do with all analytics changes, especially something that we'd be trying to fit into the last day of a sprint).

**Comment by theodur** [comment]
@TKDickson I personally haven't. Not sure if Chika has, but I can do a quick test

**Comment by theodur** [comment]
Confirmed login works on the iOS simulator

**Comment by TKDickson** [comment]
@theodur - @cadibemma also just DM'd me that she has confirmed she can log in on both platforms. 

Approved for merge to develop; no QA done by QA.

(Also I originally put this comment on the wrong PR - the backend custom errors one - in case anyone got an email about that 🤦‍♀️ )

**Review by TKDickson — APPROVED** [review]

**Pull request merged**